modules/installation-vsphere-installer-infra-requirements.adoc

```diff
@@ -475,7 +490,7 @@ ifndef::upi[]
 `VirtualMachine.Provisioning.DeployTemplate`
 
 |vSphere vCenter data center
-|If the installation program creates the virtual machine folder. For user-provisioned infrastructure, `VirtualMachine.Inventory.Create` and `VirtualMachine.Inventory.Delete` privileges are optional if your cluster does not use the Machine API. If your cluster does use the Machine API and you want to set the minimum set of permissions for the API, see the "Minimum permissions for the Machine API" table.
+|The installation program creates the virtual machine folder. If your cluster does use the Machine API and you want to set the minimum set of permissions for the API, see the "Minimum permissions for the Machine API" table.
```

Comment:
If the folder already exists it won't be created, and only in IPI

Reply:
Thanks, Joe. Updated. This section will only target IPI docs.